.github/workflows/ci.yaml

```diff
@@ -17,10 +17,16 @@ jobs:
     runs-on: ${{ matrix.platform }}
     steps:
     - name: Install Go
-      uses: actions/setup-go@v1
+      uses: actions/setup-go@v2
```

Comment:
sigh.. this was why it failed initially; https://github.com/actions/setup-go#v2

> The V2 offers:
>
> - Adds `GOBIN` to the `PATH`

Reply:
Erg. I copy pasta'd this from another repo which is why it was the old one. Nice catch!